wagtail/admin/templates/wagtailadmin/pages/listing/_modern_dropdown.html

```diff
@@ -1,24 +0,0 @@
-{% load wagtailadmin_tags i18n %}
```

Comment:
This has been renamed to `_page_header_buttons.html` (matching the template tag name, and refactored), not technically deleted.